app/views/images/show.html.erb

```diff
@@ -1,8 +1,18 @@
 <h1><%= @image.title %></h1>
+<div>
+  <% if @image.likes.include?(user_id: current_user.id) %>
```

Comment:
With the above `has_many` changes, this will become `if @image.likers.include?(current_user)`